app/controllers/users_controller.rb

```diff
@@ -1,6 +1,8 @@
 class UsersController < ApplicationController
   before_action :set_user, only: [:show]
   skip_before_action :authenticate_user!, only: :show
+# Paul added to get to user profile page to edit
+  before_action :skip_authorization
```

Comment:
can you remove this line. I have a merge that fixes the problem by authorizing show